### PR TITLE
Refs #25361 -- Added test for pickling queryset of abstract-inherited models with Meta.ordering.

### PR DESCRIPTION
**Pull request by bmispelon**
While triaging some old tickets, I found that ticket-25361 appears to have been fixed as a side-effect of another issue.

I propose adding this test based on the report and closing the ticket.

### CODE REVIEW

File: tests/queryset_pickle/models.py [hunk 1]
```diff
@@ -58,3 +58,20 @@ class SomeModel(models.Model):
...
+
+class MyEvent(AbstractEvent):
+    pass
+
+
+class ConcreteEventFK(models.Model):
+    event = models.ForeignKey('MyEvent', related_name='concrete_events', on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["event"]
```
Comment (felixxm) [inline tests/queryset_pickle/models.py]: We can drop meta ordering and add it to the queryset:
```
self.assert_pickles(c.concrete_events.order_by('event))
```

File: tests/queryset_pickle/models.py [hunk 2]
```diff
@@ -58,3 +58,20 @@ class SomeModel(models.Model):
...
+    class Meta:
+        abstract = True
+        ordering = ['title']
+
+
+class MyEvent(AbstractEvent):
+    pass
+
+
+class ConcreteEventFK(models.Model):
```
Comment (felixxm) [inline tests/queryset_pickle/models.py]: We can chop `FK`.

File: tests/queryset_pickle/models.py [hunk 3]
```diff
@@ -58,3 +58,20 @@ class SomeModel(models.Model):
...
+    event = models.ForeignKey('MyEvent', related_name='concrete_events', on_delete=models.CASCADE)
```
Comment (felixxm) [inline tests/queryset_pickle/models.py]: `related_name` is not related with this issue, so I think we can drop it and use a generic name.

File: tests/queryset_pickle/tests.py [hunk 4]
```diff
@@ -238,6 +238,13 @@ def test_unsupported_unpickle(self):
         with self.assertRaisesMessage(RuntimeWarning, msg):
             pickle.loads(pickle.dumps(qs))
 
+    def test_abstract_inheritance(self):
+        g = Group.objects.create(name='test')
+        c = MyEvent.objects.create(title='test event', group=g)
+        c.concrete_events.create()
+
```
Comment (felixxm) [inline tests/queryset_pickle/tests.py]: Chop blank line.

**Comment by bmispelon** [comment]
Thanks for the review :rocket: 

I've made the changes you requested (I changed the name of the `ConcreteEventFK` model to `Edition` to try and make things a bit clearer) and double-checked that the added test actually fails in the `1.8.X` branch.

I've also amended the commit message to the format you used for the title of this pull request.

**Comment by bmispelon** [comment]
Thanks everyone! :white_flower:

**Pull request merged**